analyzers/its/expected/Net5/Net5--net5.0-S3981.json

```
@@ -2,7 +2,7 @@
 "issues":  [
 {
 "id":  "S3981",
-"message":  "The 'Count' of 'IEnumerable<T>' is always '>=0', so fix this test to get the real expected behavior.",
+"message":  "The 'Count' of 'IEnumerable<T>' is always 'True', so fix this test to get the real expected behavior.",
```

Like this: (I won't commit this myself as that would make Burgr unhappy)
```suggestion
"message":  "The 'Count' of 'IEnumerable<T>' always evaluates as 'True' regardless the size.",
```